client/src/Components/Post.test.jsx

```diff
@@ -0,0 +1,20 @@
+
+import { render, screen } from '@testing-library/react';
+import Post from './Post';
+
+test('renders blog post', () => {
+  const samplePost = {
+    img_url: 'sampleImageUrl',
+    title: 'Sample Title',
+    content: 'This is a sample content for testing.'
+  };
+
+  render(<Post posts={samplePost} />);
+
+  const titleElements = screen.getAllByText('Sample Title');
+  const contentElements = screen.getAllByText('This is a sample content for testing.');
```

Comment:
```suggestion
  const titleElements = screen.getAllByText(samplePost.title);
  const contentElements = screen.getAllByText(samplePost.content);
```

Don't manually retype values that you are testing for; this leaves room for errors or breakage over time if the content changes